### PR TITLE
Document read only inputs

### PR DESCRIPTION
**Pull request by timja**
See https://github.com/jenkinsci/kubernetes-plugin/pull/811#discussion_r466604802

### Proposed changelog entries

* Entry 1: Issue, Human-readable Text
* ...

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [ ] (If applicable) Jira issue is well described
- [ ] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change). [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  * Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [ ] Appropriate autotests or explanation to why this change has no tests
- [ ] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers

@mention

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are correct
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins-ci.org/issues/?filter=12146)).

### CODE REVIEW

File: core/src/main/resources/lib/form/checkbox.jelly [hunk 1]
```diff
@@ -46,6 +46,10 @@ THE SOFTWARE.
     <st:attribute name="negative" />
     <st:attribute name="readonly">
       If set to true, this will take precedence over the onclick attribute and prevent the state of the checkbox from being changed.
+      
+      Note: if you want an actual read only checkbox then add:
+      <j:set var="readOnlyMode" value="true"/> inside your entry tag
```
Comment (jglick) [inline core/src/main/resources/lib/form/checkbox.jelly:51]:
```suggestion
      &lt;j:set var="readOnlyMode" value="true"/&gt; inside your entry tag
```

and similarly in other files

File: core/src/main/resources/lib/form/textbox.jelly [hunk 2]
```diff
@@ -28,6 +28,8 @@ THE SOFTWARE.
     Generates an input field <tt>&lt;input type="text" ... /></tt> to be
     used inside &lt;f:entry/>
 
+    For a read only input set <j:set var="readOnlyMode" value="true"/> inside your entry tag
```
Comment (timja) [inline core/src/main/resources/lib/form/textbox.jelly:31]:
```suggestion
    For a read only input set &lt;j:set var="readOnlyMode" value="true"/&gt; inside your entry tag
```

File: core/src/main/resources/lib/form/radio.jelly [hunk 3]
```diff
@@ -30,6 +30,9 @@ THE SOFTWARE.
     Note that safari doesn't support onchange.
 
     Beware that the name attribute should be uniquified among all radio blocks on the page, such as by prefixing it with "G0025." or whatever gensym.
+
+    For a read only radio input set <j:set var="readOnlyMode" value="true"/> inside your entry tag
```
Comment (timja) [inline core/src/main/resources/lib/form/radio.jelly:34]:
```suggestion
    For a read only radio input set &lt;j:set var="readOnlyMode" value="true"/&gt; inside your entry tag
```

**Review by jglick — APPROVED** [review]
:heart:

**Review by res0nance — APPROVED** [review]

**Review by stellargo — APPROVED** [review]

**Pull request merged**